tools/mpy_ld.py

```diff
@@ -437,6 +437,8 @@ def populate_got(env):
             dest = ".data.rel.ro"
         elif got_entry.sec_name.startswith(".bss"):
             dest = ".bss"
+        elif got_entry.sec_name == ".external.mp_port_fun_table":
+            dest = 1000 + got_entry.sym.port_fun_offset
```

Comment:
Instead of using 1000 I'd suggest something like: `dest = ("port_fun_table", got_entry.sym.port_fun_offset)`